From ecdcc6e06ef2ea3e2aa39ddaf498731874155317 Mon Sep 17 00:00:00 2001
From: psykose <alice@ayaya.dev>
Date: Mon, 9 Jan 2023 06:58:23 +0000
Subject: [PATCH] ui: import strings.h and config.h

strings.h required for strncasecmp, fixes an implicit function
declaration

config.h needed for the actual configuration, it sets -D_GNU_SOURCE that
is needed for strsep.
---
 src/curses/ui_manager.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/curses/ui_manager.c b/src/curses/ui_manager.c
index 15a89d1..2358a84 100644
--- a/src/curses/ui_manager.c
+++ b/src/curses/ui_manager.c
@@ -26,9 +26,12 @@
  * @brief Source of functions defined in ui_manager.h
  *
  */
+#include "config.h"
+
 #include <ctype.h>
 #include <stdio.h>
 #include <string.h>
+#include <strings.h>
 #include <math.h>
 #include <stdlib.h>
 #include <locale.h>
-- 
2.39.0

